themes/src/main/resources/theme/keycloak.v2/account/src/app/content/signingin-page/SigningInPage.tsx

```diff
@@ -122,105 +136,163 @@ class SigningInPage extends React.Component<SigningInPageProps, SigningInPageSta
                     categoryMap.set(container.type, container);
                 });
 
-                this.setState({credentialContainers: allContainers});
-                console.log({allContainers})
-            });
+                this.setState({ credentialContainers: allContainers });
+                console.log({ allContainers });
+            }
+        );
     }
 
     private handleRemove = (credentialId: string, userLabel: string) => {
-      this.context!.doDelete("/credentials/" + credentialId)
-        .then(() => {
+        this.context!.doDelete("/credentials/" + credentialId).then(() => {
             this.getCredentialContainers();
-            ContentAlert.success('successRemovedMessage', [userLabel]);
+            ContentAlert.success("successRemovedMessage", [userLabel]);
         });
-    }
-
-    public static credElementId(credType: CredType, credId: string, item: string): string {
-        return `${credType}-${item}-${credId.substring(0,8)}`;
+    };
+
+    public static credElementId(
+        credType: CredType,
+        credId: string,
+        item: string
+    ): string {
+        return `${credType}-${item}-${credId.substring(0, 8)}`;
     }
 
     public render(): React.ReactNode {
         return (
-            <ContentPage title="signingIn"
-                     introMessage="signingInSubMessage">
-                <Stack gutter='md'>
-                    {this.renderCategories()}
-                </Stack>
+            <ContentPage title="signingIn" introMessage="signingInSubMessage">
+                {this.renderCategories()}
             </ContentPage>
         );
     }
 
     private renderCategories(): React.ReactNode {
-        return (<> {
-            Array.from(this.state.credentialContainers.keys()).map(category => (
-                <StackItem key={category} isFilled>
-                    <Title id={`${category}-categ-title`} headingLevel={TitleLevel.h2} size='2xl'>
-                        <strong><Msg msgKey={category}/></strong>
-                    </Title>
-                    <DataList aria-label='foo'>
-                        {this.renderTypes(this.state.credentialContainers.get(category)!)}
-                    </DataList>
-                </StackItem>
-            ))
-
-        }</>)
+        return (
+            <>
+                {" "}
+                {Array.from(this.state.credentialContainers.keys()).map(
+                    (category) => (
+                        <PageSection key={category} variant={PageSectionVariants.light}>
+                            {this.renderTypes(category!)}
+                        </PageSection>
+                    )
+                )}
+            </>
+        );
```

Comment:
Can be simplified to:
```suggestion
         return Array.from(credentialContainers.keys()).map(
              (category) => (
                  <PageSection key={category} variant={PageSectionVariants.light}>
                      {this.renderTypes(category!)}
                  </PageSection>
              )
          );
```